client/state/comments/selectors.js

```diff
@@ -76,9 +72,15 @@ export const getPostOldestCommentDate = createSelector(
  * @return {Immutable.Map<CommentId, CommentNode>} comments tree in the form of immutable map<CommentId, CommentNode>, and in addition a children array
  */
 export const getPostCommentsTree = createSelector(
-	( state, siteId, postId, status ) => {
+	( state, siteId, postId/*, status*/ ) => {
```

Comment:
Why do we not need `status`? Wouldn't a common view be for example, return all approved comments?